prow/pjutil/abort.go

```diff
@@ -0,0 +1,75 @@
+package pjutil
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
+)
+
+// prowClient a minimalistic prow client required by the aborter
+type prowClient interface {
+	//ReplaceProwJob updates the prow job
```

Comment:
replaces? update vs replace is a specific semantic to k8s

Reply:
I updated the comment. 

Initially, the method was `Update` but it got changed when I rebased on current master which has the new client reverted. 

My plan is to update it again when the new prow client is back into the master.